MapboxNavigation/Geometry.swift

```diff
@@ -315,3 +315,12 @@ extension CLLocation {
         return closestCoordinate.distance < maximumDistance
     }
 }
+
+
+/*
+ Returns the smallest angle between two angles
+ */
+func smallestAngle(alpha: Double, beta: Double) -> Double {
```

Comment:
Name this method `differenceBetweenDirections(_:_:)`. Also, use `CLLocationCoordinate2D` instead of `Double`.